gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/text.tag

```diff
@@ -24,7 +24,7 @@
 	if (value == null) {
 		value = "";
 	}
-	value = org.apache.commons.lang.StringEscapeUtils.escapeHtml(value);
```

Comment:
Hmm - I wonder if this is correct here (where is this dependency resolved when running the copy of the ActiveMQ "admin" webapp ?

As a separate PR, I wonder if we can remove this test admin webapp (and use docker container instead) ?

Reply:
Jira issue - https://issues.alfresco.com/jira/browse/MM-842